hw2/03-form.js

```diff
@@ -1,3 +1,113 @@
+nameLabel.textContent = 'Name*';
+nameLabel.className = 'form-label';
+nameInput.type = 'text';
+//A name must be entered.
+nameInput.required = 'true';
+nameInput.className = 'form-control';
+//Attach label to the input for accessibility.
+nameInput.id = 'name';
+nameLabel.htmlFor = 'name';
+//Append the name label and input to the section.
+section1.append(nameLabel);
+section1.append(nameInput);
+//Append the section to the form.
+form.append(section1);
+
+//Second section for email input.
+section2.className = 'mb-3';
+emailLabel.textContent = 'Email*';
+emailLabel.className = 'form-label';
+emailInput.type = 'email';
+//Email must be entered.
+emailInput.required = 'true';
+emailInput.className = 'form-control';
+//Attach label to the input for accessibility.
+emailInput.id = 'email';
+emailLabel.htmlFor = 'email';
+section2.append(emailLabel);
+section2.append(emailInput);
+form.append(section2);
+
+//Third section to submit a message.
+section3.className = 'mb-3';
+messageLabel.textContent = 'Submit your message:';
+messageLabel.className = 'form-label';
+messageInput.className = 'form-control';
+//Attach label to the input for accessibility.
+messageInput.id = 'message';
+messageLabel.htmlFor = 'message';
+section3.append(messageLabel);
+section3.append(messageInput);
+form.append(section3);
+
+//Fourth section for checkbox.
+section4.className = 'mb-3';
+checkboxLabel.className = 'form-check-label ms-2';
+checkboxLabel.textContent = 'Sign up for the newsletter';
+checkboxInput.type = 'checkbox';
+checkboxInput.className = 'form-check-input';
+checkboxInput.id = 'check';
+checkboxLabel.htmlFor = 'check';
+section4.append(checkboxInput);
+section4.append(checkboxLabel);
+form.append(section4);
+
+//Fifth section for buttons. Using Bootstrap grid system.
+section5.className = 'mb-3 row';
+submitButton.className = 'ms-3 col btn btn-primary';
+submitButton.type = 'submit';
+submitButton.textContent = 'Submit';
+resetButton.className = 'ms-1 me-3 col btn btn-secondary';
+resetButton.type = 'reset';
+resetButton.textContent = 'Reset';
+section5.append(submitButton);
+section5.append(resetButton);
+form.append(section5);
+
+//EventListener when the submit button is pressed.
+form.addEventListener('submit', handleSubmit);
+
+//Function to handle when submit button is pressed.
+function handleSubmit(event) {
+  event.preventDefault();
+  console.log('========= Form Submission =========');
+  console.log('Name: ' + nameInput.value);
+  console.log('Email: ' + emailInput.value);
+  //Check if feedback input is empty to display alternative message.
+  if (!messageInput.value) console.log('No feedback was submitted.');
+  else console.log('Feedback: ' + messageInput.value);
+  //Check if checkbox is checked or unchecked.
+  if (checkboxInput.checked == true)
+    console.log('Newsletter: Yes, I would like to join the newsletter.');
+  else if (checkboxInput.checked == false)
+    console.log('Newsletter: No, thank you.');
+}
```

Comment:
I think it is cool you only altered the js files. I do not have much to comment on this beyond that.